File: python/003.py
#!/usr/bin/env python

import sys
import string

f1 = open("col1.txt", "w")
f2 = open("col2.txt", "w")

for line in sys.stdin:
    cols = line.split("\t")
    f1.write(cols[0] + "\n")
    f2.write(cols[1])

File: ruby/003.rb
#!/usr/bin/env ruby
#

f1 = open( "col1.txt", "w" )
f2 = open( "col2.txt", "w" )

STDIN.read.split( "\n" ).each do |r|
  c = r.split( "\t" )
  f1.puts( c[0] )
  f2.puts( c[1] )
end
